backend/src/services/course-category/course-category.service.ts

```diff
@@ -0,0 +1,20 @@
+import { CourseCategoryGetResponseDto } from '~/common/types/types';
+import { courseCategory as courseCategoryRep } from '~/data/repositories/repositories';
+
+type Constructor = {
+  courseCategoryRepository: typeof courseCategoryRep;
+};
+
+class CourseCategory {
+  #courseCategoryRepository: typeof courseCategoryRep;
+
+  constructor({ courseCategoryRepository }: Constructor) {
+    this.#courseCategoryRepository = courseCategoryRepository;
+  }
+
+  async getByName(name: string): Promise<CourseCategoryGetResponseDto | null> {
+    return this.#courseCategoryRepository.getByName(name);
+  }
```

Comment:
```suggestion
 getByName(name: string): Promise<CourseCategoryGetResponseDto | null> {
    return this.#courseCategoryRepository.getByName(name);
  }
```
more details [https://github.com/BinaryStudioAcademy/bsa-2022-guruhub/pull/103#discussion_r945107162](https://github.com/BinaryStudioAcademy/bsa-2022-guruhub/pull/103#discussion_r945107162)